.github/workflows/release.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
                   git config --local user.name "Github Actions"
                   git checkout -b ${{ inputs.releaseVersion }}-release
                   mvn -B release:clean release:prepare release:perform -Possrh-release -DreleaseVersion=${{ inputs.releaseVersion }} -DdevelopmentVersion=${{ inputs.developmentVersion }} -Darguments="-Dhabushu.usePyenv=false"
```

Comment:
I (Informational): updated based on some post-release processing that didn't work in the GitHub Action

Reply:
Just curious what was the issue here? Do we need to update our other technology brewery release scripts as well?